src/App.jsx

```diff
@@ -14,11 +14,9 @@ function App() {
     <Routes>
       <Route path="/" element={<Header />} exact></Route>
       <Route path="/allArticles" element={<AllArticles />}></Route>
-      <Route path="/allArticles/:article_id" element={<Article />}></Route>
-      <Route
-        path="/allArticles/:article_id/comments"
-        element={<Comments />}
-      ></Route>
+      <Route path="/allArticles/:article_id" element={<Article />}>
+        <Route path="comments" element={<Comments />} />
```

Comment:
Would make more sense in my opinion to include comments via the individual article page as all comments are made in relation to a specific article. Up to you/how you've planned your app but should features should match up to your API. In short - you don't need a separate route/page for comments necessarily